docs/concepts/services-networking/ingress.md

```diff
@@ -273,7 +273,7 @@ You can achieve the same by invoking `kubectl replace -f` on a modified Ingress
 
 ## Failing across availability zones
 
-Techniques for spreading traffic across failure domains differs between cloud providers. Please check the documentation of the relevant Ingress controller for details. Please refer to the federation [doc](/docs/user-guide/federation/) for details on deploying Ingress in a federated cluster.
+Techniques for spreading traffic across failure domains differs between cloud providers. Please check the documentation of the relevant Ingress controller for details. Please refer to the federation [doc](/docs/concepts/cluster-administration/federation.md) for details on deploying Ingress in a federated cluster.
```

Comment:
To preempt the redirect, you should leave off the `.md` and end it w/ a forward-slash, so `/docs/concepts/cluster-administration/federation/`.